bottlerocket-settings-models/settings-extensions/kubernetes/src/lib.rs

```diff
@@ -93,14 +99,7 @@ pub struct KubernetesSettingsV1 {
     cluster_domain: DNSDomain,
     node_ip: IpAddr,
     pod_infra_container_image: SingleLineString,
```

Comment:
The comment on line 89 wasn't correct, so I re-organized these to fix it and also removed the `Generated in x` comments when its now in all versions due to version deprecation.